pypeit/scripts/collate_1d.py

```diff
@@ -81,7 +81,7 @@ def get_report_metadata(object_header_keys, spec_obj_keys, file_info):
         header = file_info.spec1d_header_list[i]
 
         # Get the spec1d header metadata needed for the report
-        # Use getattr for the spec_obj data because one of the attributes is actually a property (med_s2n)
+        # NO LONGER: Use getattr for the spec_obj data because one of the attributes is actually a property (med_s2n)
         spec_obj_data = [getattr(spec_obj, x) for x in spec_obj_keys]
```

Comment:
Perhaps for @kbwestfall : Since this PR moves the S/N value into a datamodel entry, everything in `spec_obj` is now an attribute without any properties.  Is there a better way to produce the `spec_obj_data` list in this case?

Reply:
This approach is still fine.  Because of the way the `DataContainer` class is constructed, this approach is almost identical to, e.g., `[spec_obj[x] for x in spec_obj_keys]`.

Reply:
I think you can remove the comment.